skypy/supernova/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+This module contains methods that model the intrinsic properties of supernova.
```

Comment:
Typo (Ia)
```suggestion
This module contains methods that model the intrinsic properties of supernovae.
```

Reply:
Sure! I got confused when you said "supernova" singular